src/functional/ffront/type_info.py

```diff
@@ -286,6 +312,37 @@ def is_concretizable(symbol_type: ct.SymbolType, to_type: ct.SymbolType) -> bool
     return False
 
 
+def _is_empty_field(field: ct.SymbolType) -> bool:
```

Comment:
```suggestion
def _is_zero_dimensional_field(field: ct.SymbolType) -> bool:
```
Same for the rest.

Reply:
this name is a bit too long IMO, how about `_is_zero_dim_field`?